src/commands/scan-iac.yml

```diff
@@ -0,0 +1,14 @@
+description: Scan your applications IaC files for issues
+parameters:
+  path:
+    description: Path to a specific file or directory to test
+    type: string
+    default: ""
+steps:
+  - run:
+      name: Scan for IaC issues
+      environment:
+        SNYK_INTEGRATION_NAME: CIRCLECI_ORB
+        SNYK_INTEGRATION_VERSION: REPLACE_ORB_VERSION
+      command: >
+        snyk iac test <<#parameters.path>> <<parameters.path>><</parameters.path>>
```

Comment:
We could remove the `iac test` part and make the whole argument list into a single parameter to support any CLI command. It would avoid needing to support/document arguments twice.

Reply:
I'd argue for doing this as part of the 1.0 release. What I'm trying to do here is a tactical effort to allow IaC, not a breaking change on the Orbs usage (given I dont own the orb long term, making a change like that seems too much?)

Reply:
We could add a "command" parameter with "test' by default to keep it backwards compatible?

Reply:
https://github.com/snyk/snyk-orb/pull/43

What about this?